.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+jobs:
+  lint:
+    docker:
+      - image: hashicorp/terraform:0.12.2
+    steps:
+      - checkout
+      - run:
+          name: Lint
+          command: terraform fmt -recursive -check=true -diff=true
+  tag:
+    machine: true
+    parameters:
+      version:
+          description: "Module version"
+          type: string
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "bd:5a:bf:66:f2:6a:cf:d4:45:32:4b:19:1a:db:87:2d"
```

Comment:
Why did you need to include this?

Reply:
I implemented auto-tagging via CircleCI, since Terraform Registry finds new versions of a module via git annotated tags that conform to semver. The deploy key that CircleCI uses is read-only, so I could not push the annotated tag. I created my own read/write deploy key instead, as otherwise I'd need to create a machine user or use one of my personal keys (no thnx) with significant access to our private repositories. The deploy key grants r/w access to this repository only. I needed to add the fingerprint so CircleCi will pick up the key and use it.